app/views/tags/_row.html.erb

```diff
@@ -1,15 +1,15 @@
 <div class="mb-3 row plain-container">
   <div class="col-sm-2">
-    <%= form.label :email, 'Tag', class: 'col-form-label' %>
+    <%= form.label :tag, 'Tag', class: 'col-form-label' %>
```

Comment:
Should this be `:name` as we don't have a field with a name of `:tag` on this page.

Reply:
Fixed.